src/main/java/duke/Event.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ *  This class represents an Event.
+ *  An event: tasks that start at a specific time and ends at a specific time.
+ *
+ * @author Ryan Tian Jun.
+ */
+public class Event extends Task {
+
+    private String at;
+    private LocalDate date;
+
+    /**
+     * This constructor handles the creation af a new Event Task.
+     *
+     * @param description Task description.
+     * @param type Task type: Event.
+     * @param by Event timeline.
+     */
+    public Event(String description, Type type, String by) {
```

Comment:
very small issue, but maybe consider changing input String by to at? Less confusing and more consistent

Reply:
Agreed! I have changed it to at.